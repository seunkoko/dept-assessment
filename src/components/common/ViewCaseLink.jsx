// eslint-disable-next-line
import React from 'react';
import { Link } from 'react-router-dom';

import './common.scss';

const ViewCaseLink = ({
  id,
  moreDescription,
}) => {
	return (
    <>
      <Link
        className="btn view-case-link"
        data-toggle="collapse"
        role="button"
        to={`#${id}`}
        data-target={`#${id}`}
        aria-expanded="false"
        aria-controls={id}
      >
        <i className="fas fa-caret-right mr-2" />
        View Case
      </Link>

      <div className="collapse more-description" id={id}>
        <div className="">{ moreDescription }</div>
      </div>
    </>
	);
};

export default ViewCaseLink;
