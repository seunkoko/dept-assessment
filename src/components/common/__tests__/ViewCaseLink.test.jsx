import React from 'react';

import { shallow } from 'enzyme';

import ViewCaseLink from '../ViewCaseLink';

describe('ViewCaseLink', () => {
	const component = shallow(<ViewCaseLink />);

	it('should render ViewCaseLink component', () => {
		expect(component).toMatchSnapshot();
  });
  
});