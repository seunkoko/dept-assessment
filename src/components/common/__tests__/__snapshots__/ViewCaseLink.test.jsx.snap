// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ViewCaseLink should render ViewCaseLink component 1`] = `
<Fragment>
  <Link
    aria-expanded="false"
    className="btn view-case-link"
    data-target="#undefined"
    data-toggle="collapse"
    role="button"
    to="#undefined"
  >
    <i
      className="fas fa-caret-right mr-2"
    />
    View Case
  </Link>
  <div
    className="collapse more-description"
  >
    <div
      className=""
    />
  </div>
</Fragment>
`;
